update examples/ directory (#146)
C2Devel/terraform-provider-croccloud: examples/eip/README.md

# Elastic IP Example

The EIP example launches the nginx web server and creates a security group.

> This example assumes an SSH key pair was created in the cloud console.

Reviewer: ```suggestion
This example assumes that an SSH key pair was created in the cloud console.
```

Running the example:

```
$ export AWS_ACCESS_KEY_ID="your-access-key"
$ export AWS_SECRET_ACCESS_KEY="your-secret-key"
$ terrafom init
$ terraform apply -var="key_name=your-key-name"
```

> In a few minutes the nginx welcome page will be available at the elastic IP address.

Reviewer: ```suggestion
In a few minutes, you can reach the nginx welcome page at the Elastic IP address.
```

Reviewer: с глаголами оно как-то душевнее
